dataplane/Dockerfile

```
@@ -1,26 +1,42 @@
-FROM archlinux as builder
+FROM ubuntu:23.10 as builder
 
-RUN pacman -Syu --noconfirm
-RUN pacman -S base-devel protobuf rustup --noconfirm
+ARG TARGETARCH
 
+RUN apt update
+RUN apt install --yes build-essential protobuf-compiler curl pkg-config llvm
```

`apt` (as opposed to `apt-get`) is "meant for end-users" and doesn't guarantee backwards compatibility, so it is generally preferable to use the lower-level `apt-get`:

```suggestion
RUN apt-get update
RUN apt-get install --yes build-essential protobuf-compiler curl pkg-config llvm
```